contracts/src/v0.8/dev/ConfirmedOwner.sol

```diff
@@ -1,79 +1,22 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+import "./ConfirmedOwnerWithProposal.sol";
+
 /**
  * @title The ConfirmedOwner contract
  * @notice A contract with helpers for basic contract ownership.
  */
-contract ConfirmedOwner {
-
-  address private s_owner;
-  address private s_pendingOwner;
-
-  event OwnershipTransferRequested(
-    address indexed from,
-    address indexed to
-  );
-  event OwnershipTransferred(
-    address indexed from,
-    address indexed to
-  );
-
-  constructor(address newOwner) {
-    s_owner = newOwner;
-  }
+contract ConfirmedOwner is ConfirmedOwnerWithProposal {
```

Comment:
I'm guessing this is for backwards compatibility with the stuff already using ConfirmedOwner - couldn't we just update all the 0.8 contracts to use ConfirmedOwnerWithProposal (hard cutover to simplify)?

Reply:
https://github.com/smartcontractkit/chainlink/pull/4632#discussion_r660654129